blueprints/ember-frost-core/index.js

```diff
@@ -1,35 +1,33 @@
+const blueprintHelper = require('ember-frost-core/blueprint-helper')
+
 module.exports = {
-  afterInstall: function () {
-    const packagesToRemove = [
-      'ember-frost-button',
-      'ember-frost-checkbox',
-      'ember-frost-css-core',
-      'ember-frost-icons',
-      'ember-frost-link',
-      'ember-frost-loading',
-      'ember-frost-password',
-      'ember-frost-select',
-      'ember-frost-text',
-      'ember-frost-textarea',
-      'ember-frost-theme',
-      'svg4everybody'
-    ].map((packageName) => {
-      return {name: packageName}
-    })
+  afterInstall: function (options) {
+    const addonsToAdd = [
+      {name: 'ember-computed-decorators', target: '~0.2.0'},
+      {name: 'ember-concurrency', target: '~0.7.15'},
+      {name: 'ember-elsewhere', target: '~0.4.1'},
+      {name: 'ember-hook', target: '^1.4.1'},
+      {name: 'ember-prop-types', target: '^3.0.0'},
+      {name: 'ember-spread', target: '^1.1.1'},
+      {name: 'ember-truth-helpers', target: '^1.2.0'}
+    ]
 
-    return this.removePackagesFromProject(packagesToRemove)
-      .then(() => {
+    // Get the packages installed in the consumer app/addon. Packages that are already installed in the consumer within
+    // the required semver range will not be re-installed or have blueprints re-run.
+    const consumerPackages = blueprintHelper.consumer.getPackages(options)
+
+    // Get the packages to install (not already installed) from a list of potential packages
+    return blueprintHelper.packageHandler.getPkgsToInstall(addonsToAdd, consumerPackages).then((pkgsToInstall) => {
+      if (pkgsToInstall.length !== 0) {
+        // Call the blueprint hook
         return this.addAddonsToProject({
-          packages: [
-            {name: 'ember-concurrency', target: '~0.7.15'},
-            {name: 'ember-computed-decorators', target: '~0.2.0'},
-            {name: 'ember-elsewhere', target: '~0.4.1'},
-            {name: 'ember-frost-test', target: '^1.0.0'},
-            {name: 'ember-prop-types', target: '^3.0.0'},
-            {name: 'ember-truth-helpers', target: '^1.2.0'}
-          ]
+          packages: pkgsToInstall,
+          blueprintOptions: {
+            saveDev: true
```

Comment:
In `ember-cli@2.8` they remove the default behavior so we need to specify this.
https://github.com/ember-cli/ember-cli/issues/6318

They put the default behavior back in `ember-cli@2.11`